packages/desktop-client/src/components/Titlebar.js

```diff
@@ -159,6 +159,68 @@ export function SyncButton({ localPrefs, style, onSync }) {
   );
 }
 
+export function OutdatedButton() {
+  async function fetchJSON(...args) {
+    let res = await fetch(...args);
+    let json = await res.json();
+    return json;
+  }
+
+  function cmpSemanticVersion(a, b) {
+    let x = a
+      .replace('v', '')
+      .split('.')
+      .map(n => parseInt(n));
+    let y = b
+      .replace('v', '')
+      .split('.')
+      .map(n => parseInt(n));
+
+    return x[0] - y[0] || x[1] - y[1] || x[2] - y[2];
+  }
+
+  let [latestVersion, setLatestVersion] = useState('');
+  let [clientVersion, _] = useState(`v${window.Actual.ACTUAL_VERSION}`);
+  useEffect(() => {
+    (async () => {
+      let tags = (
+        await fetchJSON('https://api.github.com/repos/actualbudget/actual/tags')
+      )
+        .map(t => t.name)
+        .concat([clientVersion]);
+      tags.sort(cmpSemanticVersion);
+      console.log(tags);
+      setLatestVersion(tags[0]);
+    })();
+  }, []);
+
+  let [showTooltip, setShowTooltip] = useState(false);
+  return (
+    latestVersion !== '' &&
+    cmpSemanticVersion(clientVersion, latestVersion) !== 1 && (
+      <Button
+        bare
+        onClick={_ => {
+          window.open('https://actualbudget.github.io/docs', '_blank');
```

Comment:
Maybe this could link to the release notes page?

Reply:
I linked it to https://actualbudget.com/release-notes/ for now, as it is the one linked in the docs and https://actualbudget.com/blog/?filter=release does not seem to work. Is there a more recent link that I should use instead?